zip-0227.rst

```diff
@@ -291,10 +291,13 @@ The following is a list of rationale for different decisions made in the proposa
 
 TxId Digest
 ===========
-A new issuance transaction digest algorithm is defined that constructs the identifier for an issuance transaction. Each branch of the tree of hashes will correspond to a specific subset of transaction data. The overall structure of the hash is as follows; each name referenced here will be described in detail below::
+
+As in ZIP 244 [#zip-0244]_, the digests are all personalized BLAKE2b-256 hashes, and in cases where no elements are available for hashing, a personalized hash of the empty byte array is used.
+
+A new issuance transaction digest algorithm is defined that constructs the identifier for an issuance transaction. Each branch of the tree of hashes will correspond to a specific subset of issuance transaction data. The overall structure of the hash is as follows; each name referenced here will be described in detail below::
 
     issuance_txid_digest
-    ├── issueActions
+    ├── issue_actions_digest
     └── issuerVerificationKey
```

Comment:
They both should have the same case: `issue_actions` or `issueActions` not both.

Reply:
The naming convention in ZIP 244 is at the bottom of [this](https://zips.z.cash/zip-0244#txid-digest) section -- `camelCase` if it is not a hash of something else, `snake_case`

I see an inconsistency in the use of both `issuerVerificationKey` and `issuanceValidatingKey` which I am now fixing.